### DISCUSSION TITLE
Is femr.datasets removed? Alternative for preparing a patient database with OMOP data?

### DISCUSSION THREAD

**Discussion by jordanhilsman** [som-shahlab/femr#192] [Q&A]
Hi. I want to create a patient database similar to tutorial 2a in FEMRv1, but with FEMR0.2.3 femr.datasets no longer exists as an import. Should I revert to an older version of femr for this functionality?

**Comment by EthanSteinberg** [comment]
Patient databases have been replaced by HuggingFace datasets that store patients in MEDS format (see https://github.com/Medical-Event-Data-Standard/)

Looking through the README again, I can see why this was confusing. I'll add some text to better describe the desired data input format.

If you already have a patient database (from an earlier version of FEMR, feel free to continue using FEMR v1)

**Reply by EthanSteinberg** [reply]
If you explicitly want to work with OMOP data https://github.com/Medical-Event-Data-Standard/meds_etl/blob/main/src/meds_etl/omop.py is an ETL from OMOP into MEDS
